multibase.csv

```diff
@@ -5,17 +5,19 @@ base8,             7,    octal,
 base10,            9,    decimal,                                                  draft
 base16,            f,    hexadecimal,                                              default
 base16upper,       F,    hexadecimal,                                              default
-base32hex,         v,    rfc4648 no padding - highest char,                        candidate
-base32hexupper,    V,    rfc4648 no padding - highest char,                        candidate
-base32hexpad,      t,    rfc4648 with padding,                                     candidate
-base32hexpadupper, T,    rfc4648 with padding,                                     candidate
-base32,            b,    rfc4648 no padding,                                       default
-base32upper,       B,    rfc4648 no padding,                                       default
-base32pad,         c,    rfc4648 with padding,                                     candidate
-base32padupper,    C,    rfc4648 with padding,                                     candidate
+base32hex,         v,    rfc4648 case-insensitive - no padding - highest char,     candidate
+base32hexupper,    V,    rfc4648 case-insensitive - no padding - highest char,     candidate
+base32hexpad,      t,    rfc4648 case-insensitive - with padding,                  candidate
+base32hexpadupper, T,    rfc4648 case-insensitive - with padding,                  candidate
+base32,            b,    rfc4648 case-insensitive - no padding,                    default
+base32upper,       B,    rfc4648 case-insensitive - no padding,                    default
+base32pad,         c,    rfc4648 case-insensitive - with padding,                  candidate
+base32padupper,    C,    rfc4648 case-insensitive - with padding,                  candidate
 base32z,           h,    z-base-32 (used by Tahoe-LAFS),                           draft
-base58flickr,      Z,    base58 flicker,                                           candidate
+base36,            k,    base36 [0-9a-z] case-insensitive - no padding,            default
+base36upper,       K,    base36 [0-9a-z] case-insensitive - no padding,            default
```

Comment:
```suggestion
base36upper,       K,    base36 [0-9a-z] case-insensitive - no padding,            draft
```

(need wide implementation to be "default", even if we're planning on making it a default)

Reply:
All fixed. Let's keep it open for a day until the rest of the stuff lands.

If anyone **REALLY** dislikes `K`: **SPEAK NOW!** 😜